includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -13,7 +13,7 @@
  */
 class AMP_Allowed_Tags_Generated {
 
-	private static $spec_file_revision = 734;
+	private static $spec_file_revision = 751;
```

Comment:
If we update this to 753 then we'll get `@page` rule support.

Reply:
Sounds good, @westonruter. Would you like to hold this PR until revision `753` [is released](https://github.com/ampproject/amphtml/tags), and not merge this PR for `1.0-RC2`?

It looks like the master branch [is at revision 756](https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L29), though that's not released in a tag yet.